manifests/base/rstudio-buildconfig.yaml

```diff
@@ -0,0 +1,80 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  annotations:
+    opendatahub.io/notebook-image-creator: RHOAI
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
+    opendatahub.io/notebook-image-name: "R Studio"
+    opendatahub.io/notebook-image-desc: "R Studio workbench image, allows to integrated development environment for R, a programming language for statistical computing and graphics."
```

Comment:
@grainnejenningsRH We should copyedit this, the line shows in the UI, this way

![image](https://github.com/red-hat-data-services/notebooks/assets/442720/01f84fbe-bfef-411a-97d2-b18d3e6a47a6)

(note the sentence is here twice, first as image description (below in the PR, the screenshot shows this) and here as buildconfig description.

Reply:
Python v3.9? Who cares about Python version in an R image?

Reply:
Great observation, probably will need to switch to R version

Reply:
I didn't got it about the text... The text which is on the imageStream annotation is the text that the UI demonstrates

Reply:
Ohhh okay, i saw it now! I will fix that, thanks

Reply:
I will use that `The R Studio Workbench image enables the integration of a development environment for R, a programming language designed for statistical computing and graphics.` WDYT?

Reply:
Could mirror what we have in some of the other image labels and use 'with', if that would be accurate here. e.g.:

`R Studio Workbench image with an integrated development environment for R, a programming language designed for statistical computing and graphics.`

Reply:
Good enough for this PR, I suppose, it is correct English now.

There is still an opportunity for a new subsequent PR that would look at all the labels of all the images and adjust them all together, to give the product some consistency, though. My objection is essentially this.

The usecase is, "as a user, I want to start R workbench in my browser and do some data science". So I have the workbench spawner opened, and I see this text. The phrasing "enables the integration of a development environment" absolutely does not fit into the context of this usecase. I want to see something like "The R Studio Workbench image starts R studio in the browser". I don't have any sort of "integration" in mind at that point.

But, that is a different task, "Minor UI text updates and improvements". To be performed later.

Reply:
Thank you folks, I changed the text as per Grainne proposal. I agree Jiri, this can be another one task on the notebooks "Cleaning"!